### PR TITLE
Hiveminds cant get objectives to target other hiveminds

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
title

## Why It's Good For The Game
you cant assimilate other hosts so its impossible

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
tweak: hiveminds cant get objectives to target another host
/:cl:

### CODE REVIEW

**Comment by etherware-novice** [comment]
alright unironically how do you test this? (i stg kat if you add a testing evidence tag)

**Comment by itsmeow** [comment]
Use dream daemon to run a server, connect to localhost via BYOND's Open, then logout and connect again as guest.

Once in game use admin player panel to add hivemind datum to both clients.

**Comment by itsmeow** [comment]
Also I'm not really sure if this is a bug or not, it seems there are objectives to assimilate other hives. @Penwin0 ?

**Comment by etherware-novice** [comment]
> alright unironically how do you test this? (i stg kat if you add a testing evidence tag)

@itsmeow can you read

**Comment by Fronsis** [comment]
> Also I'm not really sure if this is a bug or not, it seems there are objectives to assimilate other hives. @Penwin0 ?

I think the wording might be wrong on the title, assimilate other hive is when you full-gib them(i think) the ''bug'' comes from when your objective is to make another hivemind host an awakened vessel.

Round ID: 40698 I was a hivemind host, and another person (Bungo) had the objective to make me into an ''awakened vessel'' 

In order to do that you need to yellow grab the vessel and after a bit you ''awakened'' them, Bungo tried it on me (hive mind host) and it didn't work, so yeah i'd say there needs to be a check on that particular objective to no target other hiveminds hosts since you can't do it, you can normally do it with regular crewmembers that you assimilated.

**Comment by Penwin0** [comment]
> Also I'm not really sure if this is a bug or not, it seems there are objectives to assimilate other hives. @Penwin0 ?

The one being fixed here is infact an oversight on my part

File: code/game/gamemodes/hivemind/objectives.dm [hunk 1]
```diff
@@ -72,6 +72,11 @@
 	else
 		explanation_text = "Free Objective"
 
+/datum/objective/hivemind/awaken/is_valid_target(datum/mind/possible_target)
+	. = ..()
+	if(possible_target.has_antag_datum(/datum/antagonist/hivemind))  // you cant assimilate other hosts for some reason
```
Comment (Penwin0) [inline code/game/gamemodes/hivemind/objectives.dm]: Bad comment

**Review by Penwin0 — APPROVED** [review]

**Pull request merged**